compiler/qsc_rca/src/scaffolding.rs

```diff
@@ -0,0 +1,296 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use crate::{
+    common::{GlobalSpecId, SpecKind},
+    ApplicationsTable, CallableComputeProperties, ComputePropertiesLookup, ItemComputeProperties,
+    PackageComputeProperties, PackageStoreComputeProperties,
+};
+use qsc_data_structures::index_map::IndexMap;
+use qsc_fir::fir::{
+    BlockId, ExprId, LocalItemId, PackageId, PackageStore, StmtId, StoreBlockId, StoreExprId,
+    StoreItemId, StoreStmtId,
+};
+
+/// Scaffolding used to build the package store compute properties.
+#[derive(Debug, Default)]
+pub struct PackageStoreScaffolding(IndexMap<PackageId, PackageScaffolding>);
```

Comment:
Given that all of these types are in the scaffolding module, it is redundant to suffix the types with the moddule name which already prefixes them.

Reply:
Renamed the data structures in this module that had the redundant suffix.